### DISCUSSION TITLE
Is there an inbuild function to combine transformations for SE3 objects. Something like multiplying two transformation matrixes

### DISCUSSION THREAD

**Discussion by nea98** [facebookresearch/theseus#600] [Q&A] [answered]
I want to multiply two SE3 objects. Is there an inbuild function for that?

**Comment by mhmukadam** [comment, marked as answer]
Do you mean [compose](https://github.com/facebookresearch/theseus/blob/main/theseus/geometry/lie_group.py#L142)?
